package.json

```diff
@@ -1,11 +1,15 @@
 {
-  "name": "forsetti-ah-frontend",
+  "name": "authors-haven",
   "version": "1.0.0",
-  "description": "Authors Haven - A Social platform for the creative at heart",
+  "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --config ./webpack.config.js --mode development --open",
```

Comment:
Good job @BukkyOmo, I think adding the hot reloading option will be cool too (--hot) for webpack to push the changes to the browser without leaving the editor, what do you think?

Reply:
I already configured that in the webpack.config.js where I specified hot as true, therefore even if I do not add --hot to the package.json scripts, the browser still loads itself automatically without loss of state. Check out this link to understand it better https://gaearon.github.io/react-hot-loader/getstarted/